lecture/views.py

```
@@ -0,0 +1,56 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
```

Ruff (0.8.2)

`django.contrib.messages` imported but unused

Remove unused import: `django.contrib.messages`

(F401)